perf/load/loadclient/values.yaml

```diff
@@ -2,7 +2,7 @@
 # This should be done on the command line by --set ingress=<ip_addres>
 
 connections: 8
-qps: 50
+qps: 300
```

Comment:
Did you mean to check this in? I think its fine, we should probably test higher traffic